recipe/meta.yaml

```diff
@@ -87,3 +78,4 @@ about:
 extra:
   recipe-maintainers:
     - mikofski
+    - xylar
```

Comment:
@mikofski, I'm hoping you're okay with adding me as a maintainer.

Reply:
Yes, it’s great to get help. I actually don’t even use Duluth much anymore so if you want to take over this and fast export I would appreciate greatly!

Reply:
I added myself on fastimport as well.